health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Hypothesis.js

```
@@ -173,11 +173,11 @@ const Hypothesis = ({ category, assumptions: initialAssumptions,setShowToast,all
         {showPopUP && <PopUp
           className={"popUpClass"}
           type={"default"}
-          heading={t("CONFIRM_TO_DELETE")}
+          heading={t("HYP_CONFIRM_TO_DELETE")}
           equalWidthButtons={true}
           children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)